core/common-ui/build.gradle.kts

```
@@ -0,0 +1,17 @@
+plugins {
+    alias(libs.plugins.team.noweekend.android.library)
+    alias(libs.plugins.team.noweekend.android.compose)
+}
+
+android {
+    namespace = "team.noweekend.core.common.ui"
+}
+
+dependencies {
+    implementation(project(":core:common-android"))
+    implementation(project(":core:common-kotlin"))
```

```suggestion
    implementation(project(":core:common-android"))
    implementation(project(":core:common-kotlin"))
    implementation(project(":core:model"))
```
p2 : 이부분은 다같이 논의가 필요해보이는데용
common-ui에서 model에 있는 도메인 모듈을 직접 바라보게 할건지, 아니면 클린아키텍처 관점에 맞게 data, domain, ui 모델을 별도로 설계할지 논의 필요해보입니당.

---

클린 아키텍처 기반으로  코드를 작성하긴 했는데, 나는 피처 모듈쪽이 ui레이어랑 가깝게 생각되어서, 진짜 common-ui는 컴포넌트를 모아둔 느낌으로 생각했거든? 

그래서 model을 바라봐도 괜찮다고 생각하긴해!

---

클린아키텍처 기반이라면 feature모듈에서 ui model을 만들어서 사용하면 될 것 같고, common-ui에선 그럼 예외적으로 ui model이 아니라 domain model을 허용하는 것으로 이해되는데 맞을까욥~?
이방식도 상관없을 것 같긴해요~